include/platform/mir/input/event_builder.h

```diff
@@ -46,13 +49,15 @@ class EventBuilder
         xkb_keysym_t keysym,
         int scan_code) = 0;
 
+    [[deprecated("use the pointer_event() that includes all properties instead")]]
     virtual EventUPtr pointer_event(
         std::optional<Timestamp> timestamp,
         MirPointerAction action,
         MirPointerButtons buttons_pressed,
         float hscroll_value, float vscroll_value,
         float relative_x_value, float relative_y_value) = 0;
```

Comment:
Is there any point to retaining these APIs?

Reply:
As far as I know we're not breaking platform ABI

Reply:
This isn't an ABI we're expecting outside folks to use yet. So breaking it isn't a big deal